src/CSS/TestModal.css

```diff
@@ -0,0 +1,235 @@
+/** Modal Styling **/
```

Comment:
I would probably include a comment linking to the source of this CSS

Reply:
True, true. I just forgot.